### ISSUE TITLE
LC0016 false positive on groups inside a Promoted area

### ISSUE THREAD

**Issue by Arthurvdv** [StefanMaron/BusinessCentral.LinterCop#496]
```AL
page 50100 MyPage
{
    actions
    {
        area(Processing)
        {
            action(MyAction) { }
        }
        area(Promoted)
        {
            group(Category_Process) // Caption is missing.
            {
                actionref(MyAction_Promoted; MyAction) { }
            }
        }
    }
}
```

It is possible to set an `Caption` property on the `group(Category_Process)`, but I believe that this isn't used and/or displayed in the webclient. So exclude groups inside a Promoted area for this rule.
